src/shared/components/Animations/ListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import './list-item.less';
+
+export interface ListItemProps {
+  label: string;
+  projectNumber?: number;
+  description?: string;
+  logo?: string;
+  deprecated?: boolean;
```

Comment:
instead of having a `deprecated` prop, I'd create a `tags` props. An array of [tags](https://ant.design/components/tag/) to display

Reply:
I forgot to change the props from the old component!

I'll keep everything generic here